lib/krane/cluster_resource_discovery.rb

```diff
@@ -23,7 +23,8 @@ def global_resource_kinds
     private
 
     def fetch_globals
-      raw, _, st = kubectl.run("api-resources", "--namespaced=false", output: "wide", attempts: 5)
+      raw, _, st = kubectl.run("api-resources", "--namespaced=false", output: "wide", attempts: 5,
+        use_namespace: false)
```

Comment:
I'd prefer not to push this logic into the kubectl class it would be very special purpose. As an example of a future thing we might want: to list all api-resources we'd still want `use_namespace: false` but not to add `--namespaced=`. So it would still require a second param.